### PR TITLE
[innawood] less naked NPCs

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary
Mods "[innawood] less naked NPCs"

#### Purpose of change

Realism, consistency

#### Describe the solution

Finds the last few random NPC classes and blocks items. Updates default item groups to fit with innawoods, assigns them to NPC classes as appropriate. Completely overhauls gun item groups that may be assigned to NPCs to only allow blunderbuss and flintlocks.

#### Describe alternatives you've considered

More nudity, less nudity, more leather goods, more and less pottery and seeds

#### Testing

Game loads no errors, NPCs spawn and look right. Looks like there is still one or more melee weapon group getting pulled by the default NPCs that need a followup PR

Screen Shot 2022-07-21 at 1 14 47 PM

#### Additional context

“The clearest way into the Universe is through a forest wilderness.”  - John Muir

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
I don't think any smokeless powder, cartridge-using guns would be available in the woods. Where would the smokeless powder & casings have come from 🤔

File: data/mods/innawood/itemgroups/guns.json [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+    "items": [
+      { "item": "ashot", "prob": 100 },
+      { "item": "surv_hand_cannon", "prob": 100, "charges-min": 0, "charges-max": 5 },
+      { "item": "surv_six_shooter", "prob": 100, "charges-min": 0, "charges-max": 6 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "guns_smg_common",
+    "//": "Semi-auto civilian SMGs owned by citizens and found in many locations.",
```
Comment (TheShadowFerret) [inline data/mods/innawood/itemgroups/guns.json:40]: Also none of the weapons on here are semi-auto now 😛

Comment (TheShadowFerret) [reply data/mods/innawood/itemgroups/guns.json:36]: ~~I demand (x)bow groups reee~~

Yes

**Comment by Light-Wave** [comment]
If you haven't already, consider replacing modern guns with ranged weapons such as slings, sling staffs, bows, crossbows, and maybe even slingshots. I'm not sure how well NPCs deal with throwing weapons such as javelins, but they could also be worth including if they work well.

File: data/mods/innawood/itemgroups/guns.json [hunk 2]
```diff
@@ -262,13 +262,13 @@
   {
     "type": "item_group",
     "id": "sidearms_cop",
-    "items": [ { "group": "guns_pistol_improvised", "prob": 20, "charges": [ 0, 1 ] } ]
+    "items": [ { "group": "null", "prob": 20 } ]
```
Comment (TheShadowFerret) [inline data/mods/innawood/itemgroups/guns.json]: No flintlock pistol?

Comment (LyleSY) [reply data/mods/innawood/itemgroups/guns.json]: Tried, but it was throwing pocket errors. This group is for going into holsters that don't fit a long flintlock pistol.

**Pull request merged**